docs/usage/configuration-options.md

```diff
@@ -177,13 +177,20 @@ Example use:
 
 This setting is only applicable if you opt in to configure `automerge` to `true` for any of your dependencies.
 
-Automerging defaults to using Pull Requests (`automergeType="pr"`).
+Automerging defaults to merging Pull Requests by Renovate itself.
 In that case Renovate first creates a branch and associated Pull Request, and then automerges the PR on a subsequent run once it detects the PR's status checks are "green".
 If by the next run the PR is already behind the base branch it will be automatically rebased, because Renovate only automerges branches which are up-to-date and green.
 If Renovate is scheduled for hourly runs on the repository but commits are made every 15 minutes to the main branch, then an automerge like this will keep getting deferred with every rebase.
 
 Note: if you have no tests but still want Renovate to automerge, you need to add `"requiredStatusChecks": null` to your configuration.
 
+With `pr-auto` you can speed up the automerge process by using platform automerge facilities. Setting this will configure PRs to be merged after all (if any) branch policies have been met. Currently available for Azure and GitLab.
+
+You can also configure this using `packageRules` if you want to use it selectively (e.g. per-package).
+
+**Caution for GitLab < 12.7**: when this option is enabled it is possible due to a bug in GitLab that MRs with failing pipelines might still get merged.
```

Comment:
```suggestion
**Do not enable `pr-auto` when using GitLab < 12.7**: enabling `pr-auto` can lead to Merge Requests getting merged even though there are _failing_ pipelines.
```

I think the takeaway: "Do not enable when using old GitLab", should be the bolded part. 😉